packages/components/src/components/spinner/__snapshots__/spinner-styles.spec.ts.snap

```diff
@@ -453,13 +453,13 @@ exports[`getComponentCss() should return correct css for size: "small" and theme
 svg {
   display: block;
   fill: none;
-  animation: rotate var(--p-animation-duration, 2s) steps(50) infinite;
+  animation: rotate var(--p-motion-duration, 1.2s) linear infinite;
 }
 circle:first-child {
   stroke: #6B6D70;
 }
 circle:last-child {
-  animation: dash var(--p-animation-duration, 2s) steps(50) infinite;
+  animation: dash var(--p-motion-duration, 1.2s) linear infinite;
```

Comment:
This got almost twice as fast. Is this intended?

Reply:
Yes, it is intended, like here in the design examples: https://porsche-design-system.github.io/ux-prototype/motion/index.html